binding/python/README.md

```diff
@@ -30,15 +30,15 @@ pip3 install pvrecorder
 Getting the list of input devices does not require an instance:
 
 ```python
-from pvrecorder import PvRecorder
+from project import PvRecorder
```

Comment:
we are not importing from project are we?

Reply:
no, that must've been a global refactor change